test/sched.py

```diff
@@ -53,13 +55,20 @@
 
 def set_PBS_env():
     """ Set up the environment to recreate being in a hpc job """
-    pbsnodefile = tempfile.NamedTemporaryFile(delete=False)
+    if not os.path.isdir('/tmp/%s' % os.environ['USER']):
+        os.mkdir('/tmp/%s' % os.environ['USER'])
+    pbsnodefile = tempfile.NamedTemporaryFile(dir='/tmp/%s' % os.environ['USER'], delete=False)
     pbsnodefile.write("localhost\nlocalhost\n")
     pbsnodefile.close()
     os.environ['PBS_NODEFILE'] = pbsnodefile.name
+    os.chmod(pbsnodefile.name, stat.S_IRUSR)
+    os.chmod(os.path.dirname(pbsnodefile.name), stat.S_IRUSR|stat.S_IXUSR)
+
 
 def cleanup_PBS_env(orig_env):
     """ cleanup the mock job environment """
+    os.chmod(pbsnodefile.name, stat.S_IWUSR)
+    os.chmod(os.path.dirname(pbsnodefile.name), stat.S_IWUSR|stat.S_IXUSR)
     os.remove(os.environ['PBS_NODEFILE'])
```

Comment:
aah, I overlooked that this actually removes to temporary file, together with resetting the environment >_<

so, we should revert the changes made in #107 where we drop the use of `cleanup_PBS_env` to ensure that the temporary file is cleaned up properly